SayTheirNames/Source/Utils/Images/STNImage.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  STNImages.swift
+//  SayTheirNames
+//
+//  Created by Juan Olivera on 6/4/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+import Kingfisher
+
+struct STNImage {
```

Comment:
`enum` is more common for namespacing as it doesn't have an `init()`

Reply:
what do you think about having an extension on `UIImageView` instead?

Reply:
I ended up doing an extension now it will be just setting an image with: 
`personImageView.populate(withURL: url)`